src/dbcp/settings.yaml

```diff
@@ -1,12 +1,12 @@
-- generation_num: 1741396059395591
+- generation_num: 1742928593170421
   metadata:
-    schema_generation_number: '1741396058325017'
+    schema_generation_number: '1742928586963059'
     table_id: tblu7b4Yj58Iq2xCF
   name: airtable/Offshore Wind Locations Synapse Version/Locations.json
   pinned: false
-- generation_num: 1741396058788973
+- generation_num: 1742928592428151
   metadata:
-    schema_generation_number: '1741396058325017'
+    schema_generation_number: '1742928586963059'
```

Comment:
hmmm im not sure if these should be updated or not? I think they should? I'm changing which base the archive actually points at, but I think the system of updating the generation number stands.